src/ledger/NetworkConfig.cpp

```diff
@@ -328,16 +328,20 @@ initialCpuCostParamsEntry(Config const& cfg)
 }
 
 ConfigSettingEntry
-initialStateExpirationSettings()
+initialStateExpirationSettings(Config const& cfg)
 {
     ConfigSettingEntry entry(CONFIG_SETTING_STATE_EXPIRATION);
 
     entry.stateExpirationSettings().autoBumpLedgers =
         InitialSorobanNetworkConfig::AUTO_BUMP_NUM_LEDGERS;
     entry.stateExpirationSettings().maxEntryExpiration =
         InitialSorobanNetworkConfig::MAXIMUM_ENTRY_LIFETIME;
+
+    // TESTING_MINIMUM_PERSISTENT_ENTRY_LIFETIME defaults to
+    // InitialSorobanNetworkConfig::MINIMUM_PERSISTENT_ENTRY_LIFETIME
     entry.stateExpirationSettings().minPersistentEntryExpiration =
-        InitialSorobanNetworkConfig::MINIMUM_PERSISTENT_ENTRY_LIFETIME;
+        cfg.TESTING_MINIMUM_PERSISTENT_ENTRY_LIFETIME;
```

Comment:
It would be a bit cleaner to have this as an optional set to none. But not a big deal either way.